spec/mailers/request_access_mailer.erb_spec.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+def mock_presenter
```

Comment:
move this into your `before do` block below. Also add `RequestAccessMailer.request_access(presenter).deliver_now` because you're calling it each time as well.